RazorEngineCore/RazorEngineCompiledTemplate.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
 using System.IO;
 using System.Reflection;
+using System.Threading.Tasks;
 
 namespace RazorEngineCore
 {
     public class RazorEngineCompiledTemplate
     {
         private readonly MemoryStream assemblyByteCode;
-        private readonly Type templateType;
+        protected readonly Type templateType;
```

Comment:
Forgot I made this protected, might want to change it back in the next release! 😞

Reply:
no worries, I'll fix that